### ISSUE TITLE
Sulfur Trioxide does not craft in Large Chemical Reactor in 2.5.4

### ISSUE THREAD

**Issue by felix-schneider** [GregTechCEu/GregTech#1596]
### GregTech CEu Version

2.5.4

### Modpack Used

Nomifactory GTCEu 1.5.2

### Environment

Singleplayer

### New Worlds

Yes

### Cross-Mod Interaction

Unsure

### Expected Behavior

Expected sulfur trioxide to produce in the LCR.

### Actual Behavior

The recipe does not run. It works in a regular chem reactor, as do other recipes using non-consumed catalysts. Just this recipe does not work.

### Steps to Reproduce

Build a large chem reactor.
Insert inputs. Nothing happens

**Comment by felix-schneider** [comment]
Sorry, I just realized this recipe has been edited in Nomifactory, so the bug should be there.

**Comment by felix-schneider** [comment]
After discussion in https://github.com/tracer4b/nomi-ceu/issues/284
I am reopening this issue, as it seems to be a GTCEu issue.

Note that the SO3 recipe was edited by crafttweaker:
```
// H2SO4 require V2O5
// Sulfur Trioxide * 1000
<recipemap:chemical_reactor>.findRecipe(7, null, [<liquid:sulfur_dioxide> * 1000, <liquid:oxygen> * 1000]).remove();
<recipemap:large_chemical_reactor>.findRecipe(7, null, [<liquid:sulfur_dioxide> * 1000, <liquid:oxygen> * 1000]).remove();

chemical_reactor.recipeBuilder()
    .fluidInputs(<liquid:sulfur_dioxide> * 1000, <liquid:oxygen> * 1000)
    .notConsumable(<metaitem:dustVanadiumPentoxide>)
    .fluidOutputs(<liquid:sulfur_trioxide> * 1000)
    .duration(200).EUt(7).buildAndRegister();
```

However, other recipes by crafttweaker which work exactly the same (chem reactor recipes, using a catalyst, replacing an original GTCEu recipe) work. As far as I know, this is the only one which doesn't work.

Could be related to #1600 or #1574 ?

**Comment by ALongStringOfNumbers** [comment]
This should now be fixed as of #1633
